website/docs/getting_started/flashing_firmware/flashing_firmware_cli.md

```diff
@@ -0,0 +1,77 @@
+---
+id: flashing_firmware_cli
+title: Flashing Firmware – CLI
+sidebar_label: Command Line Interface
+slug: /getting_started/flashing_firmware/cli
+---
+
+:::caution
+Be very careful to install the correct load for your board. In particular the popular 'T-BEAM' radio from TTGO is not called 'TTGO-Lora' (that is a different board). So don't install the 'TTGO-Lora' build on a TBEAM, it won't work correctly.
+:::
+
+## Linux
+
+Requires `Python` and `pip`
+
+1. Download and unzip the latest Meshtastic firmware [release](https://github.com/meshtastic/Meshtastic-esp32/releases).
+2. `pip install --upgrade esptool` - Installs esptool on your machine
+3. Connect your radio to your USB port
+4. `esptool.py chip_id` - Confirm that your device is talking to your PC by running
+You should see something like:
+```
+mydir$ esptool.py chip_id
+esptool.py v2.6
+Found 2 serial ports
+Serial port /dev/ttyUSB0
+Connecting....
+Detecting chip type... ESP32
+Chip is ESP32D0WDQ6 (revision 1)
+Features: WiFi, BT, Dual Core, 240MHz, VRef calibration in efuse, Coding Scheme None
+MAC: 24:6f:28:b5:36:71
+Uploading stub...
+Running stub...
+Stub running...
+Warning: ESP32 has no Chip ID. Reading MAC instead.
+MAC: 24:6f:28:b5:36:71
+Hard resetting via RTS pin...
+```
+5. `cd` into the directory where you unzipped the latest release.
+6. Install the correct firmware for your board with `device-install.sh -f firmware-BOARD-VERSION.bin`
+```
```

Comment:
Set language for codeblocks ```bash
also, try to remove prompt's from commands:

Not recommended:
```
mydir$ esptool.py chip_id
```

Preferred:
```bash
esptool.py chip_id
```

Reply:
I understand. I'm reworking this section completely. However, I believe that since we are modeling the output that the user should expect the `mydir$` or at least a `$` should be represented. I'll put something together today that is more clearly a command and an expected output. Above there are commands mixed with notes, so I'll fix that too. This is based off the existing documentation located at [Meshtastic-device](https://github.com/meshtastic/Meshtastic-device/#installing-from-a-commandline)

Reply:
Yeah, I'd like different color code blocks for output and commands, maybe highlighting, also lots of the old docs are hacked together, so take the formatting with a grain of salt